extension/src/popup/basics/Toggle/index.tsx

```diff
@@ -0,0 +1,72 @@
+import React, { cloneElement, useEffect, useState } from "react";
+import "./styles.scss";
+
+enum LabelPosition {
+  left = "left",
+  right = "right",
+}
+
+interface ToggleComponent {
+  labelPosition: typeof LabelPosition;
+}
+
+interface ToggleProps extends React.InputHTMLAttributes<HTMLInputElement> {
+  id: string;
+  checked: boolean;
+  customInput?: React.ReactElement;
```

Comment:
@quietbits What do you think about this `customInput` implementation for `Toggle`? I want to be able to pass it a Formik Input (which automatically tracks state change

Reply:
I think it works nicely. :+1: